Yumemi-ios-training/Yumemi-ios-training/Presenter/WeatherPresenter.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  WeatherPresenter.swift
+//  Yumemi-ios-training
+//
+//  Created by 工藤 海斗 on 2022/03/24.
+//
+
+import Foundation
+
+protocol WeatherPresenterProtocolInput {
+    func feachWeather()
+}
+
+protocol WeatherPresenterProtocolOutput: AnyObject {
+    func showWeather(weaherType: WeatherType)
+    func showErrorAlert(errorMessage: String)
```

Comment:
[FYI]
実引数名、仮引数名を付けると呼び出し側が見やすくなったりします！
以下一例:

```suggestion
protocol WeatherPresenterProtocolOutput: AnyObject {
    func showWeather(of weather: WeatherType)
    func showErrorAlert(with message: String)
```